### ISSUE TITLE
stats file compatibility

### ISSUE THREAD

**Issue by AlesDU** [eik3/loxgraph#2]
Hi Eike, looks like Loxone stats changed file format (pls. see attachments)...do you have some update or other solutions for yours loxgraph? Thx Alex

![0c91cbe4-039e-c206-ffff9811ae6e5944 01511](https://cloud.githubusercontent.com/assets/9684265/10911350/b9aec9c0-8244-11e5-95f3-af804064950e.jpg)
![screen](https://cloud.githubusercontent.com/assets/9684265/10911368/c9f03e72-8244-11e5-91e9-3daabb5493e0.png)

**Comment by eik3** [comment]
Thanks Alex for trying out loxgraph!
Could you give me a screenshot of the Web Inspector Console opened in Safari? It looks like loxgraph stopped loading a file, it would be helpful to see the URL and return code. You can of course blur IDs or IPs in the screenshot.

**Comment by netdata-be** [comment]
I there an update on this one, I looks like loxone switched to binary statics, when downloaden the stats they are NOT in XML format

**Comment by jvanaert** [comment]
+1

**Comment by aleqcz** [comment]
What version of LoxoneConfig do you use? I'm on 8.3 and it works fine. Although the stats are saved in binary format (as seen on FTP), accessing them via http://<loxone>/stats/<UID>.xml gives up the XML. Eik's LoxGraph works with that.

**Comment by AlesDU** [comment]
Hi I’m using LoxoneConfig in last version = 9.1.10.20
Please let me know if LoxGraph will works with that.
A.





6. 3. 2018 v 18:42, Aleš Berka <notifications@github.com<mailto:notifications@github.com>>:


What version of LoxoneConfig do you use? I'm on 8.3 and it works fine. Although the stats are saved in binary format (as seen on FTP), accessing them via http:///stats/.xml gives up the XML. Eik's LoxGraph works with that.

**Comment by aleqcz** [comment]
Hi Aleš,
I can't tell, sorry. I'm intentionally on 8.x and I do not want to upgrade to 9.x.

Try opening http://loxone_ip/stats/ and click any entry. Do you get XML like
`<Statistics Name="Aktuální výkon M2" NumOutputs="1" Outputs=";0;0;%.1f"><S T="2017-08-01 00:00:00" V="30.000"/><S T="2017-08-01 00:10:00" V="30.000"/><S T="2017-08-01 00:20:00" V="30.000"/><S T="2017-08-01 00:30:00" V="30.000"/>
`?
If yes, then I think it should still work.

A.

**Comment by AlesDU** [comment]
Do you speak czech Aleš? Looks like they changed stat format….or? Sending you example stat file…
A.




6. 3. 2018 v 19:23, Aleš Berka <notifications@github.com<mailto:notifications@github.com>>:


Hi Aleš,
I can't let you know, I'm intentionally on 8.x and not willing to upgrade.

Try opening http://loxone_ip/stats/ and click any entry. Do you get XML like
<Statistics Name="Aktuální výkon M2" NumOutputs="1" Outputs=";0;0;%.1f"><S T="2017-08-01 00:00:00" V="30.000"/><S T="2017-08-01 00:10:00" V="30.000"/><S T="2017-08-01 00:20:00" V="30.000"/><S T="2017-08-01 00:30:00" V="30.000"/>?
If yes, then I think it should still work.

A.

**Comment by aleqcz** [comment]
Hello name-fellow,
yes I do speak Czech. But let's stay in English here for the good of the whole community. Not mentioning the original author with the biggest knowledge of this project is Eike from Germany (unless I'm mistaken). It would be foolish to assume he can speak Czech too.
Don't see your attachment. Try it via github.com web.

A.
P.S.: Please remove quoted text from your e-mail answers, it's polluting the thread on  [https://github.com/eik3/loxgraph/issues/2](url) :)

**Comment by AlesDU** [comment]
Sorry...and sending stat file made by LoxoneConfig ver. 9.1.10.20. Thx for some idea A.
[0ab71a50-03b9-4a09-ffff112233445566.201803.zip](https://github.com/eik3/loxgraph/files/1786271/0ab71a50-03b9-4a09-ffff112233445566.201803.zip)

**Comment by aleqcz** [comment]
That looks like the file I can see in /stats/ when accessing Miniserver via FTP. Did you download this file via link from a web page /stats/ when accessing http://your_loxone_ip/stats/ via browser?
If yes, what if you try directly http://your_loxone_ip/stats/0ab71a50-03b9-4a09-ffff112233445566.201803.xml
(again via browser)?

**Comment by AlesDU** [comment]
yes that's true I downloaded it via FTP. I don't know why I can't access it via browser.

**Comment by aleqcz** [comment]
I'm afraid nobody will be able to help you unless you say what exactly is the problem, why / how you can't access it via browser

**Comment by aleqcz** [comment]
As per discussion on http://www.vodnici.net/community, 9.0.9.26 still works. Personally don't think Loxone would change statistics format suddenly with 9.1, the problem will be elsewhere.

UPDATE: Even 9.1 confirmed to work.

**Comment by AlesDU** [comment]
thx, I try to solve browser issue first.

**Comment by eik3** [comment]
I'm on `9.1.1 (2017.11.14)` and Loxgraph works for me.
